doc/source/getting_started/index.rst

```diff
@@ -19,98 +19,21 @@ Install the latest release for use with this command:
 For developers
 ^^^^^^^^^^^^^^
 
-Installing the ``pyconverter-xml2py`` package in developer mode allows
-you to modify the source and enhance it.
+Installing the pyconverter-xml2py package in developer mode allows you to modify the source and enhance it.
+Refer to the :ref:`Contribute <ref_contributing>` section to know how to take part to the project.
```

Comment:
```suggestion
For contribution guidelines, see :ref:`Contribute <ref_contributing>` in the *PyAnsys Developer's Guide*.
```
You want to tell them why they might want to click the link before presenting the link. Also, you want to let them know where the link will take them (a topic, which is the link, in a guide for PyAnsys libraries.

Reply:
This is not in the *PyAnsys Developer's Guide*, it will redirect them to the Contribute section of the same documentation instead of duplicating how to install the package in Developer mode.

Reply:
@clatapie Ahh. My mistake! I removed the part where a guide is mentioned as this isn't necessary for links to other topics within the same documentation!